crates/indexer/src/geyser/programs/candy_machine.rs

```diff
@@ -138,15 +138,18 @@ mod tests {
             "candy_machines/AoHidoffmkL4xURViNgbA4YyeDw82FAYUZfomL3X5BoU.dmp",
             "candy_machines/piA76RvvmCt7UWEmJSBVA6xMoXqwvEAELwJoqeHK6i3.dmp",
             "candy_machines/CiBuYi3W3aVQbMWcjvfKBpwjHS6fViuuxQdSUUqkjkn4.dmp",
-            "candy_machines/ACDPaQ3uGy33KsBKiUH4azDX4q7Nxk3QwW3trALEdFmB.dmp", /* this one has hidden settings */
+            // This candy machine has hidden settings, but for some reason has allocated space
+            // for all the config lines that would be needed, however they are all empty.
+            // This results in returning a results array with zero as none of the config
+            // lines are marked as available
```

Comment:
just to add the the annoyingness of candy machines, some (but not all) of them seem to have erroneously allocated space for config lines, but still have hidden settings.